cmd/ponzu/generate.go

```diff
@@ -24,7 +24,7 @@ type generateField struct {
 	View     string
 }
 
-var reservedFiledNames = map[string]string{
+var reservedFieledNames = map[string]string{
```

Comment:
:disappointed: - my grade school teachers would be so disappointed! Thanks for catching this, I'll be sure to fix it on the next push.